.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -53,6 +53,7 @@ jobs:
         run: echo running !{{ ciflow_config.root_job_name }}
 {%- endif %}
   calculate-docker-image:
+    if: ${{ github.repository_owner == 'pytorch' }}
```

Comment:
These jobs are typically the first job run so they should prevent the entire workflow from running altogether

Reply:
The magic string we use on SciPy and other projects to also allow you to skip CI for just GitHub or for all jobs:
```
"github.repository == 'scipy/scipy' && !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]') && !contains(github.event.head_commit.message, '[skip github]')"
```
Maybe less of a worry here since CI resources aren't on a limited free plan, but could be useful perhaps (feel free to ignore if not).

Reply:
I believe the Github Actions dispatcher should already account for things like `[ci skip]`

Context: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

Reply:
Ah yes -fairly recent, our string is older. The `[skip github]` is still useful in SciPy et al., because things are split over GitHub Actions, Azure, TravisCI, Appveyor. Here it looks like you're good already indeed.